app/views/claims/support/support_users/check.html.erb

```diff
@@ -0,0 +1,53 @@
+<% content_for(:page_title) { t(".page_title") } %>
+
+<%= content_for(:before_content) do %>
+  <%= govuk_back_link(href: new_claims_support_support_user_path) %>
+<% end %>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= form_with(model: @user, url: claims_support_support_users_path, method: :post) do |f| %>
+        <%= f.hidden_field :first_name, value: @user.first_name %>
+        <%= f.hidden_field :last_name, value: @user.last_name %>
+        <%= f.hidden_field :email, value: @user.email %>
+
+        <label class="govuk-label govuk-label--l">
+          <span class="govuk-caption-l"><%= t(".caption") %></span>
+          <%= t(".title") %>
+        </label>
+
+        <%= govuk_summary_list do |summary_list| %>
+          <% summary_list.with_row do |row| %>
+            <% row.with_key(text: User.human_attribute_name(:first_name)) %>
+            <% row.with_value(text: @user.first_name) %>
+            <% row.with_action(text: t(".change"), href: new_claims_support_support_user_path(@user), visually_hidden_text: @user.first_name) %>
```

Comment:
The action href should include the first name, last name, email params.

Reply:
Same as above. Now addressed.